Libraries/zephyr/MAX/Include/wrap_max32_dma.h

```diff
@@ -29,8 +29,19 @@ extern "C" {
 /*
  *  MAX32665, MAX32666 related mapping
  */
-#if defined(CONFIG_SOC_MAX32665) || (CONFIG_SOC_MAX32666)
+#if defined(CONFIG_SOC_MAX32657) || (CONFIG_SOC_MAX32665) || (CONFIG_SOC_MAX32666)
 
+#if defined(CONFIG_SOC_MAX32657)
+#define ADI_MAX32_DMA_CTRL_DIS_IE MXC_F_DMA_CTRL_DIS_IE
+#define ADI_MAX32_DMA_CTRL_CTZIEN MXC_F_DMA_CTRL_CTZ_IE
+
+#define ADI_MAX32_DMA_STATUS_IPEND MXC_F_DMA_STATUS_IPEND
+#define ADI_MAX32_DMA_STATUS_BUS_ERR MXC_F_DMA_STATUS_BUS_ERR
+#define ADI_MAX32_DMA_STATUS_TO_IF MXC_F_DMA_STATUS_TO_IF
+#define ADI_MAX32_DMA_STATUS_ST MXC_F_DMA_STATUS_STATUS
+
+#define ADI_MAX32_DMA_CFG_REQ_POS MXC_F_DMA_CTRL_REQUEST_POS
```

Comment:
The MAX32657 DMA IP source is MAX32655, 
MAX32657 and MAX32655 register map and register names are exactly match, MAX32657 has two DMA this causes API function changes,

The reason we have this dma wrapper is because of there were two group MCUs, one group has multiple DMA and other group has one DMA but with new IP block. MAX32657 bring a third group which has multiple DMA and  new IP block.
So that instead of adding MAX32657 as this I prefer totally removing device groups, to simplify file content.
Please check below file.
[wrap_max32_dma.md](https://github.com/analogdevicesinc/msdk/files/15499739/wrap_max32_dma.md)